src/pe-three-tier-esc-header/widget/pe-three-tier-esc-header.u-body-html-template.html

```diff
@@ -0,0 +1,77 @@
+<nav class="navbar navbar-default" role="navigation">
+  <!-- Brand and toggle get grouped for better mobile display -->
+  <div class="navbar-row">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      <a class="navbar-home-icon" href="/{{data.urlSuffix}}"><i class="fa fa-home"></i></a>
+      <a class="navbar-brand" href="/{{data.urlSuffix}}"><img src="{{data.logo}}" class="d-inline-block align-top"  width="140" height="30"  /></a>
+    </div>
+
```

Comment:
Where possible consider using one-way binding for performance.
https://github.com/platform-experience/serviceportal-best-practice#one-way-binding